src/front/glsl/parser/types.rs

```diff
@@ -326,3 +330,54 @@ impl<'source> ParsingContext<'source> {
         })
     }
 }
+
+fn map_image_format(word: &str) -> Option<crate::StorageFormat> {
+    let format = match word {
+        // float-image-format-qualifier:
+        "rgba32f" => crate::StorageFormat::Rgba32Float,
```

Comment:
perhaps doing `use crate::StorageFormat as Sf;` at the start of the function would help?